Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2298,8 +2297,15 @@ BYE BYE BORDERS
     .z-cse.z-cse-dark #component_searchfield-inputEl,
     .z-cse.z-cse-dark #master_panel .x-hierarchy-search input[type=text],
     .z-cse.z-cse-dark #navGrid-filter-name-inputEl,
-    .z-cse.z-cse-dark #productsgrid_id-filter-prod_id-inputEl
-    { background-color: #b2afaf }
+    .z-cse.z-cse-dark #productsgrid_id-filter-prod_id-inputEl,
+    .z-cse.z-cse-dark .x-grid-filter.x-btn {
```

Comment:
Perhaps it isn't obvious, but these long lists of rules are alphabetized to an extent.

Reply:
I've changed the order of selectors

Reply:
I have changed the order of selectors